tools/topology/sof/tokens.m4

```diff
@@ -117,3 +117,11 @@ SectionVendorTokens."sof_led_tokens" {
 	SOF_TKN_MUTE_LED_USE			"1300"
 	SOF_TKN_MUTE_LED_DIRECTION		"1301"
 }
+
+SectionVendorTokens."sof_chmap_tokens" {
+	SOF_TKN_STREAM_MAP_NUM_CH_MAPS		"1400"
+	SOF_TKN_CHANNEL_MAP_INDEX		"1401"
+	SOF_TKN_CHANNEL_MAP_EXT_ID		"1402"
+	SOF_TKN_CHANNEL_MAP_MASK		"1403"
+	SOF_TKN_CHANNEL_MAP_COEFF		"1404"
```

Comment:
I am confused by this too. This suggests that sof_ch_map_tokens contains exactly one channel_map_coeff but there could be multiple ones no?

Reply:
well it is just the token id for coeff. Yes there can be var array of coeffs in var array of ch_maps (which are in stream map struct). I made an example in sof side with 2 ch_maps with 2 coeffs each...

Reply:
I will also split the coeff into a separate set of tokens just like you've done in your linux PR

Reply:
sorry I didn't get what you mean here..?

Reply:
Can you comment the tokens too - we should really be doing this for all tokens.